### PR TITLE
Support public datacenter

### PR DESCRIPTION
**Pull request by Yancey1989**
Add public datacenter

### CODE REVIEW

File: paddlecloud/paddlecloud/settings.py [hunk 1]
```diff
@@ -275,18 +275,29 @@
 #       "mount_path" "/pfs/%s/home/%s/" # mount_path % ( dc, username )
 #    }
 #}
+FSTYPE_CEPHFS = "fstype"
```
Comment (typhoonzero) [inline paddlecloud/paddlecloud/settings.py]: FSTYPE_CEPHFS = "fstype"   => FSTYPE_CEPHFS = "cephfs"

File: paddlecloud/paddlecloud/settings.py [hunk 2]
```diff
@@ -275,18 +275,29 @@
...
+FSTYPE_HOSTPATH = "hostpath"
 DATACENTERS = {
```
Comment (typhoonzero) [inline paddlecloud/paddlecloud/settings.py]: Can we change this name to "STORAGE_SERVICES" or something more meaningful?

File: paddlecloud/paddlecloud/settings.py (hunk 1 above)
Comment (Yancey1989) [inline paddlecloud/paddlecloud/settings.py]: Done.

File: paddlecloud/paddlecloud/settings.py (hunk 2 above)
Comment (Yancey1989) [inline paddlecloud/paddlecloud/settings.py]: In the configuration file of the client, we should configure the `datacenters` and the `current-datacenter` which is configured by `settings.DATACENTERS`, so maybe `STORAGE_SERVICES` only means a storage configuration?

Comment (typhoonzero) [inline paddlecloud/paddlecloud/settings.py]: Okay, leave it as it was.

**Pull request merged**